src/database/seeders/20190910075430-test-user.js

```diff
@@ -9,10 +9,22 @@ export function up(queryInterface, Sequelize) {
     username: 'johndoe',
     email: 'technitesdev@gmail.com',
     password: '123456',
```

Comment:
check as in my implementation where I generate a hash here inside this side file, hence I have "adminpassword" on .env, this will store a plain text password on the database